samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -84,6 +84,11 @@ const ContextualPopupButton = ContextualPopupDecorator(Button);
 let lastIndex = 0;
 
 class MyVirtualList extends Component {
+	constructor (props) {
+		super();
+		this.scrollMode = props.scrollMode;
```

Comment:
Why do we need to do this?

Reply:
In line 120, `props` are passed to `<div>` elem but we don't need to pass `props.scrollMode` to this `<div>`. (same problem with `args="[object Object]"` you saw in inspector)
So I deleted `props.scrollMode` in line 117.
But we need `scrollMode` value in `VirtualGridList`, so I took `scrollMode` value from `props` before deleting it.
That's why we need this code.

Reply:
:) I know what we need to do but the way you handled is not what I expected.
Could you try to use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) instead of this?
If you search in our code, you can find many usages.

Reply:
OK, I changed this to use destructuring assignment.